llvm/lib/Transforms/IPO/MergeFunctions.cpp

```diff
@@ -375,9 +375,35 @@ bool MergeFunctions::doFunctionalCheck(std::vector<WeakTrackingVH> &Worklist) {
 }
 #endif
 
+// This function iterates over two functions and compares their
```

Comment:
Outdated comment? This no longer iterates over two functions.

Reply:
Sorry I know what happened, when I updated I didn't update with my staged changes